config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
+
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
+
+## RuntimeDriver
+
+RuntimeDriver is an optional driver name to be passed to VM-based runtimes.
```

Comment:
Is the intention of `RuntimeDriver` to select between Xen, KVM, Qemu and other hypervisors?